obal/data/playbooks/scratch/metadata.obal.yaml

```diff
@@ -2,3 +2,8 @@ help: |
   Create a scratch build of a package
 
   A scratch build produces the same result as a normal release would, but it's not tagged in any repository. This allows verification of a change without actually committing it.
+variables:
+  run_koji_whitelist_check:
+    action: store_false
+    parameter: --force
```

Comment:
`force` sounds very generic to me and I'd prefer `skip-koji-check` or `ignore-koji-check` or something else that makes the parameter closer to what's actually skipped.